src/main/java/com/alexfu/sqlitequerybuilder/builder/JoinOnBuilder.java

```diff
@@ -17,7 +17,23 @@ public SelectWhereBuilder where(String condition) {
   }
 
   public SelectJoinBuilder join(String table) {
-    return new SelectJoinBuilder(this, table);
+    return new SelectJoinBuilder(this, table, JoinType.JOIN);
+  }
+
+  public SelectJoinBuilder leftOuterJoin(String table) {
+    return new SelectJoinBuilder(this, table, JoinType.LEFT_OUTER_JOIN);
+  }
+
+  public SelectJoinBuilder crossJoin(String table) {
+    return new SelectJoinBuilder(this, table, JoinType.CROSS_JOIN);
+  }
+
+  public SelectJoinBuilder naturalJoin(String table) {
+    return new SelectJoinBuilder(this, table, JoinType.NATURAL_JOIN);
+  }
+
+  public SelectJoinBuilder naturalLeftOuterJoin(String table) {
+    return new SelectJoinBuilder(this, table, JoinType.NATURAL_LEFT_OUTER_JOIN);
```

Comment:
`naturalLeftOuterJoin`, `naturalJoin`, `crossJoin`, and `leftOuterJoin` in this file can be removed. They are no longer being used anywhere. This will probably bring the code coverage back up as well.